src/main/java/gov/nist/secauto/oscal/lib/profile/resolver/support/BasicIndexer.java

```diff
@@ -327,10 +330,11 @@ protected final AbstractEntityItem.Builder newBuilder(
   /**
    * Create a new builder with the provided info.
    * <p>
-   * This method can be overloaded to support applying additional data to the returned builder.
+   * This method can be overloaded to support applying additional data to the
+   * returned builder.
    * <p>
-   * When working with identifiers that are case insensitve, it is important to ensure that the
-   * identifiers are normalized to lower case.
+   * When working with identifiers that are case insensitve, it is important to
+   * ensure that the identifiers are normalized to lower case.
```

Comment:
Apologies, I worked in this file and auto-formatting with project rules triggered this. Revert?

Reply:
Woops, disregard.